Added tests for :: trailing or leading : when checking for CWD in PATH
or LD_LIBRARY_PATH.

T. Straumann, 5/29/2012

*** support/dependencies/dependencies.sh.orig	2014-03-12 09:54:28.978235922 -0700
--- support/dependencies/dependencies.sh	2014-03-12 09:58:01.755654785 -0700
***************
*** 15,22 ****
--- 15,25 ----
  # try not to rely on egrep..
  if test -n "$LD_LIBRARY_PATH" ; then
  	echo TRiGGER_start"$LD_LIBRARY_PATH"TRiGGER_end | grep ':\.:' >/dev/null 2>&1 ||
+ 	echo TRiGGER_start"$LD_LIBRARY_PATH"TRiGGER_end | grep '::' >/dev/null 2>&1 ||
  	echo TRiGGER_start"$LD_LIBRARY_PATH"TRiGGER_end | grep 'TRiGGER_start\.:' >/dev/null 2>&1 ||
+ 	echo TRiGGER_start"$LD_LIBRARY_PATH"TRiGGER_end | grep 'TRiGGER_start:' >/dev/null 2>&1 ||
  	echo TRiGGER_start"$LD_LIBRARY_PATH"TRiGGER_end | grep ':\.TRiGGER_end' >/dev/null 2>&1 ||
+ 	echo TRiGGER_start"$LD_LIBRARY_PATH"TRiGGER_end | grep ':TRiGGER_end' >/dev/null 2>&1 ||
  	echo TRiGGER_start"$LD_LIBRARY_PATH"TRiGGER_end | grep 'TRiGGER_start\.TRiGGER_end' >/dev/null 2>&1
  	if test $? = 0; then
  		echo
***************
*** 31,38 ****
--- 34,44 ----
  # try not to rely on egrep..
  if test -n "$PATH" ; then
  	echo TRiGGER_start"$PATH"TRiGGER_end | grep ':\.:' >/dev/null 2>&1 ||
+ 	echo TRiGGER_start"$PATH"TRiGGER_end | grep '::' >/dev/null 2>&1 ||
  	echo TRiGGER_start"$PATH"TRiGGER_end | grep 'TRiGGER_start\.:' >/dev/null 2>&1 ||
+ 	echo TRiGGER_start"$PATH"TRiGGER_end | grep 'TRiGGER_start:' >/dev/null 2>&1 ||
  	echo TRiGGER_start"$PATH"TRiGGER_end | grep ':\.TRiGGER_end' >/dev/null 2>&1 ||
+ 	echo TRiGGER_start"$PATH"TRiGGER_end | grep ':TRiGGER_end' >/dev/null 2>&1 ||
  	echo TRiGGER_start"$PATH"TRiGGER_end | grep 'TRiGGER_start\.TRiGGER_end' >/dev/null 2>&1
  	if test $? = 0; then
  		echo
